doc/tutorials/create-up2-images.sh

```diff
@@ -0,0 +1,288 @@
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+
+#!/bin/bash
```

Comment:
This really should be kept as the very first line for this to be properly recognized as a bash script file.